<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDialogResize</class>
 <widget class="QDialog" name="QtDialogResize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDialogResize</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scale Size X/Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scale Size X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scale Size Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QtDialogResize</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_2</sender>
   <signal>editingFinished()</signal>
   <receiver>QtDialogResize</receiver>
   <slot>checkXYDimensionFill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_3</sender>
   <signal>editingFinished()</signal>
   <receiver>QtDialogResize</receiver>
   <slot>checkXYDimensionFill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>QtDialogResize</receiver>
   <slot>checkXYFactorFill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkXYDimensionFill()</slot>
  <slot>checkXYFactorFill()</slot>
 </slots>
</ui>
